xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -933,22 +933,20 @@ the file and the value is the label of the target it should be associated with.
 These files won't be added to the project if the target is unfocused.
 """,
         ),
-        "post_build": attr.label(
-            allow_single_file = True,
+        "post_build": attr.string(
             doc = """\
-A `Label` representing a shell script that should be run after the build.
-
-Note: Since this is referenced and run outside of the Bazel context, this
-cannot be a Bazel-generated file.
+An optional path to an excutable file that should be run after the build. You
+can specify a workspace-relative path (e.g. `./post-build.sh`), or an absolute
```

Comment:
We should just say it's the text of a script that will be run after the build. Then we can give an example (like you have) of how to easily invoke an executable (the most common case, but not the only case).